src/slices/users.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getUsers } from '../api/users';
+import { User } from '../types/User';
+
+type UsersState = {
+  users: User[];
+};
+
+const initialState: UsersState = {
+  users: [],
+};
+
+export const init = createAsyncThunk('users/fetch', () => getUsers());
```

Comment:
Consider renaming the action 'init' to something more descriptive, like 'fetchUsers'. This makes it clearer what the action does.